_source/_posts/2020-11-24-spring-boot-okta.adoc

```diff
@@ -47,7 +51,7 @@ To create a secure Spring Boot app with Okta, run `okta start spring-boot`. You'
 
 TIP: If you already have an Okta account, you can run `okta login` first.
 
-This will download our https://github.com/okta-samples/okta-spring-boot-sample[Okta Spring Boot sample], register your app on Okta, and configure it by adding your Okta settings to `src/main/resources/application.properties`.
+This will download our https://github.com/okta-samples/okta-spring-boot-sample[Okta Spring Boot sample], register your app on Okta, and configure it by adding your Okta settings to a `.okta.env` file.
```

Comment:
```suggestion
This will download our https://github.com/okta-samples/okta-spring-boot-sample[Okta Spring Boot sample], register your app on Okta, and configure it by adding your Okta settings to a `.okta.env` file.

**IMPORTANT:** Keep your secrets out of source control by adding `okta.env` to your `.gitignore` file: 

```bash
echo .okta.env >> .gitignore
```
```